assets/docs/rishit-dagli/mirnet-tfjs/default/lite/fp16/1.md

```diff
@@ -0,0 +1,32 @@
+# Tfjs rishit-dagli/mirnet-tfjs/default/lite/fp16/1
```

Comment:
I don't think the `/lite/` part is needed - it is usually used to denote that the model is TF Lite, and in this case the model is TF.js. Feel free to drop it in this files and the rest if you want to.

Reply:
Thanks for pointing that out, I will fix that. I copied the syntax part of the document from a model I had earlier published (now removed as TFHub no longer supports adding medical models) [tensorflow/hub#665](https://github.com/tensorflow/hub/pull/665) leading to which I made this and the `Module` error, sorry for that.